### DISCUSSION TITLE
Is anyone using Eclipse for Conductor development?

### DISCUSSION THREAD

**Discussion by rickfish** [Netflix/conductor#2434] [Q&A]
I have used Eclipse for Conductor development for a couple of years now. I had to 'enhance' some build.gradle files to accommodate the protogen stuff but for the most part it was straightforward.

Now with 3.x I am having a lot of problems which seem difficult to resolve. cassandra-persistence has issues with java.util being found in more than one jar, testImplementation with sourceSets.test.output in many modules does not seem to work as far as unresolved symbols that should be found, etc.

Is anyone else using Eclipse to build Conductor? I am using Eclipse 2019-12. Unfortunately I am limited to what my client has available in their software distribution library so I can't get a more recent version, not sure if that would help or not...

**Comment by Jiehong** [comment]
Hi @rickfish,

As far as I know, conductor does not suppose you're using any kind of editor / IDE at all (the CI/CD uses gradle only, and it works just fine).

This looks more like Eclipse is having issues with some gradle projects. I am not using Eclipse myself, but usually projects do not need to be changed so that it works in Eclipse / Intellij / VSCode / Sublime.
